comid/example_test.go

```diff
@@ -590,38 +590,68 @@ func Example_decode_CBOR_4() {
 }
 
 func Example_decode_CBOR_5() {
-	// TO DO Add suitable DIAG representing measurement with MKey as integer
+	// Taken from https://github.com/ietf-corim-cddl/tree/main/examples
```

Comment:
which one?  we need to be more precise here.

Reply:
Well there is no exact match now, as we are using a different Measurement Key.

As you said, I just gave a reference that the input material is taken from the examples given here!

Reply:
However, now done the change!